Frontend/src/pages/admin/CreateNotificationPage.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <q-card flat bordered id="card">
+    <q-card-section>
+      <div class="text-h6">Send Notification</div>
+    </q-card-section>
+
+    <q-separator inset />
+
+    <div class="q-pa-md">
+      <q-table
+        title="Send to"
+        :data="toSend"
+        :columns="columns"
+        row-key="studentName"
+      />
+      <q-input filled v-model="text" label="Title" style="margin-top: 3%" />
+      <q-input v-model="text" filled type="textarea" style="margin-top: 3%" />
+      <q-btn color="primary" label="Send Notification" style="margin-top: 3%" />
+    </div>
+  </q-card>
+</template>
+
+<script>
+// import HomeCurrentCoopStudentItem from 'components/admin/HomeCurrentCoopStudentItem.vue'
```

Comment:
Remove this comment if you're not gonna use this component

Reply:
this page i renamed to AdminCreateNotifPage and then made all my changes lmao not sure why this is here